aten/src/ATen/native/cuda/linalg/CUDASolver.h

```diff
@@ -7,7 +7,14 @@
 #define USE_CUSOLVER_64_BIT
 #endif
 
+#if defined(CUDART_VERSION) || defined(ROCM_VERSION) && ROCM_VERSION >= 50200
+
 #ifdef CUDART_VERSION
```

Comment:
wondering if this should be `#if defined(CUDART_VERSION) || defined(ROCM_VERSION) && ROCM_VERSION > 50200` so that when QA tests this with mainline builds, these tests will fail until hipSolver team adds the `const` keywords?

Reply:
Resolution: Only define CONST_QUALIFER to empty for ROCM_VERSION < 50300

Reply:
Makes sense. On second thought, breaking the PyTorch build itself for QA runs might not be a good strategy, since it'll prevent any sort of testing whatsoever.